app/partials/mms/printConfirm.html

```diff
@@ -24,33 +19,89 @@ <h4>{{action}} {{type | lowercase}}</h4>
+                    <input type="radio" name="export-orientation" ng-model="model.landscape" ng-value="false"> Portrait<br>
+                    <input type="radio" name="export-orientation" ng-model="model.landscape" ng-value="true"> Landscape
+                </form>
+            </div>
+            <br>
+
+            <div class="form-group">
+                <label>Header</label>
+                <div class="row narrow-gutter-row">
+                    <div class="col-md-3">
+                        <input class="ve-plain-input" type="text" ng-model="meta['top-left']" placeholder="Left" aria-labelledby="left header">
+                    </div>
+                    <div class="col-md-6">
+                        <input class="ve-plain-input" type="text" ng-model="meta.top" placeholder="Center" aria-labelledby="center header">
+                    </div>
+                    <div class="col-md-3">
+                        <input class="ve-plain-input" type="text" ng-model="meta['top-right']" placeholder="Right" aria-labelledby="right header">
+                    </div>
+                </div>
+            </div>
+            <div class="form-group">
+                <label for="footer">Footer</label>
+                <div class="row narrow-gutter-row">
+                    <div class="col-md-3">
+                        <input class="ve-plain-input" type="text" ng-model="meta['bottom-left']" placeholder="Left" aria-labelledby="left footer">
+                    </div>
+                    <div class="col-md-6">
+                        <input class="ve-plain-input" type="text" ng-model="meta.bottom" placeholder="Center" aria-labelledby="center footer">
+                    </div>
+                    <div class="col-md-3">
+                        <input class="ve-plain-input" type="text" ng-model="meta['bottom-right']"  placeholder="Right" aria-labelledby="right footer">
+                    </div>
+                </div>
+            </div>
+            <p class="help-block"><i>Use 'counter(page)' for page number</i></p>
+
+        </form>
         </div>
-        <div class="form-group">
-            <div for="footer" style="text-align:center;" >Footer:</div>
-            <input type="text" ng-model="meta['bottom-left']" size="11" placeholder="Left">
-            <input type="text" ng-model="meta.bottom" size="50" placeholder="Center">
-            <input type="text" ng-model="meta['bottom-right']" size="11" placeholder="Right">
+
+        <div class="well" ng-show="customizeDoc.useCustomStyle">
+        <form ng-submit="saveStyleUpdate()" role="form">
+
+            <div class="form-group">
+                <textarea class="form-control" rows="6" style="width:100%" ng-model="customizeDoc.customCSS"></textarea>
+            </div>
+            <button class="btn btn-default" type="submit"><i class="fa fa-save"></i>Save CSS for document<span ng-if="elementSaving"><i class="fa fa-spinner fa-spin"></i></span></button>
+
+        </form>
```

Comment:
we should add a button for previewing that pops up the print window with the css